Cargo.toml

```diff
@@ -58,6 +58,8 @@ tracing-subscriber = { version = "0.3", features = [
 
 tokio-util = "0.7.13"
 
+mimi_content = { git = "https://github.com/phnx-im/mimi-content", rev = "71495dc8fc794111c381649c4c6cbc6e258a0135" }
```

Comment:
Are we okay with git rev dependencies?

Reply:
Yes, I think that's totally fine for now.